src/components/calendar/Calendar.js

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+import FullCalendar from "@fullcalendar/react";
+import dayGridPlugin from "@fullcalendar/daygrid";
+import timeGridPlugin from "@fullcalendar/timegrid";
+import interactionPlugin from "@fullcalendar/interaction"; // needed for dayClick
```

Comment:
we can probably get rid of line 5, this is only here for something that they were doing on the demo. keeping it in just incase :)